File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/anagrafiche/ParticelleCatastaliMetodoProduzione.cs
using AgronicaCoreModelsSTD.metaschema;
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.anagrafiche
{
    public class ParticelleCatastaliMetodoProduzione
    {
        public MetodoProduzione metodoProduzione { get; set; }
        public IntervalloTemporale validita { get; set; }

        public ParticelleCatastaliMetodoProduzione()
        {

        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/anagrafiche/ParticelleCatastaliMetodoProduzione.ts
import { MetodoProduzione } from '../metaschema/MetodoProduzione';
import { IntervalloTemporale } from './IntervalloTemporale';

export class ParticelleCatastaliMetodoProduzione {
    metodoProduzione: MetodoProduzione;
    validita: IntervalloTemporale;
}
